launchable/utils/session.py

```diff
@@ -18,6 +19,25 @@ def _session_file_path(build_name: str) -> Path:
 
 
 def _get_session_id() -> str:
+    # CircleCI changes unix session id each steps, so set non change variable
+    # https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+    if os.environ.get("CIRCLECI") is not None:
+        id = os.environ.get("CIRCLE_WORKFLOW_ID")
+        if id is not None:
+             return id
+        Logger().warning("CIRCLECI environment variable is set but not CIRCLE_WORKFLOW_ID")
```

Comment:
If CIRCLECI is set but CIRCLE_WORKFLOW_ID is not set, that's an unanticipated situation so we'd be better off flagging that instead of silently returning "", which leads to undesirable behaviours.